src/translations/nbTranslations.json

```
@@ -540,6 +542,7 @@
     },
     "my_page": "Min side",
     "my_profile": {
+      "add_promoted_publication": "Legg til/fjern som favoritt",
```

key bør matche tekst
```suggestion
      "edit_promoted_publication": "Legg til/fjern som favoritt",
```